templates/dt_template/README.md

```diff
@@ -8,6 +8,10 @@
 |--------: | :-------------------- | :--------------: |
```

Comment:
I thought I left a message also about this, but do we need the table? Can't we rely on the git history? This is going to quickly rot.